src/utils/utils.go

```diff
@@ -107,3 +108,15 @@ func FindAndRemoveElementFromStringList(s []string, r string) []string {
 	}
 	return s
 }
+
+func Retry(attempts int, sleep time.Duration, log *logrus.Logger, f func() error) (err error) {
```

Comment:
maybe better to use retry from package https://github.com/avast/retry-go ?

Reply:
Don't want to import package for a 4 lines function